perl/lib/Seq/Role/Message.pm

```diff
@@ -24,8 +24,8 @@ use Carp        qw/croak/;
 use Time::HiRes qw(time);
 use Try::Tiny;
 
-my $PUBLISHER_ACTION_TIMEOUT  = 30;
-my $PUBLISHER_CONNECT_TIMEOUT = 10;
+my $PUBLISHER_ACTION_TIMEOUT  = 20;
```

Comment:
I increased these, because in the case of network issues, it is probably better to waste a few minutes than stop sending messages altogether; this also now matches the Python version.

Also, I set the ACTION_TIMEOUT (which controls the heartbeat) to be smaller than the CONNECT_TIMEOUT which controls the socket timeout, because in some socket implementations (e.g. Python) operations on the socket are subject to the socket timeout, and so it would be impossible for the heartbeat to ever run. That isn't the case for Perl, but that is an implementation detail, and we sacrifice nothing by making the connection timeout longer than the heartbeat interval.